tools/updatecli.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
```

Comment:
Might be better to use a different name than `updatecli` or it might be confusing.

Reply:
Renamed it but i was not really inspired